File: CustomRating/app/src/main/java/com/example/customrating/MainActivity.java
package com.example.customrating;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        CustomRatingView customRatingView = findViewById(R.id.customView);
        customRatingView.setMaximum(5);
        customRatingView.setStarting(2);
    }
}

File: app/src/main/java/com/example/sprint3_animatedrating/MainActivity.kt
package com.example.sprint3_animatedrating

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    val manager = supportFragmentManager

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Showfrgm_popup()
    }

    fun Showfrgm_popup() {
        val transaction = manager.beginTransaction()
        val fragment = frgm_popup()
        transaction.replace(R.id.frag_holder,fragment)
        transaction.addToBackStack(null)
        transaction.commit()




    }


}
